### ISSUE TITLE
Special-case fault declaration of whole partition without loading all sectors

### ISSUE THREAD

**Issue by anorth** [filecoin-project/specs-actors#640]
After #599, processing the fault of an entire partition (as for a missed Window PoSt) won't require loading the sector infos for all of the sectors. We should make this fast path possible for a miner to invoke explicitly when suffering a major fault. 

As well as better for our chain load, this will be necessary for a very large miner to declare the faults rather than suffer a missed PoSt, since limits on sector loading (explicit, and block gas limit) may prevent a giant miner from addressing all their sectors quickly enough to declare them in time.

**Comment by ZenGround0** [comment]
The second paragraph is out of date with v2 first-missed-post-for-free change but the first is still relevant.

**Comment by ZenGround0** [comment]
Revisiting this, I've realized that partition RecordMissedPost avoids loading all sector infos and this issue is about changing partition DeclareFaults to the same behavior.  Since DeclareFaults is not rational to call with first-missed-post-for-free change I don't think we should fix this.

@anorth please verify I'm not missing anything and reopen if I am.

**Comment by ZenGround0** [comment]
From discussion on #1278 it looks like DeclareFaults as a concept is here to stay for the long term so I'm reopening this.  However I'm going to label it as won't fix until a future time when we modify fault declaration to actually make sense for mining operators to use.
